data-mermaid/mermaid-webapp
# [487] Column tool tip

[trello card](https://trello.com/c/qRpY0zIj/487-column-header-tool-tip)

summary:

add info icon to some column headers to provide more context to users

to test:
1. refer to [mermaid terms list](https://docs.google.com/spreadsheets/d/1CpGEMW_qF7XeI52H-gvZZKM8cwhHDg6yV7eUbnyFE6M/edit?pli=1#gid=2048735771) to check out which items have added tool tips
2. go to users page (for example)
3. click on admin/collector/read-only info icon
4. observe pop up text
5. click away from tool tip to close it
